doc/install.rst

```diff
@@ -109,19 +125,25 @@ By default, DLR will be built with CPU support only. To enable support for NVIDI
 
 You will need to install NVIDIA CUDA and TensorRT toolkits and drivers beforehand.
 
+Building for OpenCL Devices
+"""""""""""""""""""""""""""
+
 Similarly, to enable support for OpenCL devices, run CMake with:
 
 .. code-block:: bash
 
   cmake .. -DUSE_OPENCL=ON 
   make -j4
 
+Install Python package
+""""""""""""""""""""""
+
 Once the compilation is completed, install the Python package by running ``setup.py``:
 
 .. code-block:: bash
 
   cd ../python
-  python3 setup.py install --user
+  python3 setup.py install --user --force
```

Comment:
Can you give an example of the case when -f is needed? (with material evidences)

Reply:
If a previous install of DLR already exists, you will need to use force otherwise it will not replace the existing one. This is because the version of DLR is still 1.0 so it thinks it’s the same version.

Reply:
Actually, it will replace it (this is why I asked for material evidences). The version is analyzed and the installation is skipped in case you use `pip` and your package needs to be downloaded from repo.
But `setup.py` does not analyze the version if the package files are on the drive.

Below are install log which shows installation and reinstallation without force. After the reinstallation all files were replaced with new one (exactly the dame size but updated time)
https://gist.githubusercontent.com/apivovarov/47715d2a804de7757388d29f95084534/raw/ed5870d6b1f91fd4e21f1141a147055ea5862276/install.log

Reply:
[March 17, 2020, 1:27 PM] Morris, Trevor: Thanks, I am fine with removing force since I am unable to reproduce the behavior I saw before